packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -24,6 +22,7 @@ import {
   hasContents,
 } from './InternalHelpers';
 import { IHttpNegotiationResult, NegotiatePartialOptions, NegotiationOptions } from './types';
+import { ProblemJsonError } from '@stoplight/prism-http';
```

Comment:
This is currently a TS error, as it cannot find it. I'm not yet familiar with the TS setup here, are we supposed to import from "ourselves" like this?
```
import { ProblemJsonError } from 'http/src'
```
fixes the problem.

Reply:
Oh I made a mistake here, fixed.